meta/main.yml

```diff
@@ -38,12 +38,16 @@ galaxy_info:
         - buster
         - bullseye
         - bookworm
+        # Docker does not yet officially support Debian Trixie.  See
+        # https://docs.docker.com/engine/install/debian/ for more
+        # details.
+        # - trixie
     # RedHat support was removed from this Ansible role because some
     # of its tasks are Debian-specific.
```

Comment:
Now that Fedora is supported again, shouldn't this comment be removed?
```suggestion
```

Reply:
Wait, it looks like Fedora is still unsupported in the molecule tests, so now I am confused.

Reply:
No RedHat-based distributions are supported.  Please see commit 5686ab8.